tests/conditional_compilation/conftest.py

```diff
@@ -171,3 +194,91 @@ def save_session_info(pytestconfig, artifacts):
     """Fixture function for saving additional attributes to configuration file."""
     yield
     write_session_info(path=artifacts / SESSION_INFO_FILE, data=pytestconfig.session_info)
+
+
+@pytest.fixture(scope="session")
+def omz_path(request):
+    """Fixture function for command-line option."""
+    omz_path = request.config.getoption("omz_repo", skip=True)
+    if omz_path:
+        try:
```

Comment:
`try except` construction is superfluous. Because `omz_path` is required arguments for correct work of tests